zerver/openapi/zulip.yaml

```diff
@@ -12127,6 +12144,17 @@ paths:
                     type: integer
                   default: []
                   example: [1, 2]
+                subscribe_to_default_streams:
+                  description: |
+                    Boolean indicating whether to subscribe the newly created user
+                    to default streams in the organization.
+
+                    This parameter cannot be passed as `true` if the `stream_ids`
+                    list is not empty.
```

Comment:
It feels a bit messy to have to mention this every time; has this detail been discussed in the API design stream?

Reply:
I don't think this has been discussed yet.